iocore/net/SSLClientUtils.cc

```diff
@@ -66,6 +66,7 @@ verify_callback(int preverify_ok, X509_STORE_CTX *ctx)
   netvc = SSLNetVCAccess(ssl);
 
   if (netvc != nullptr) {
+    netvc->callHooks(TS_EVENT_SSL_SERVER_VERIFY_HOOK);
```

Comment:
@ushachar Yah, I have a PR for making  proxy.config.ssl.client.verify.server overridable. Now I think this config should accept another value to indicate that we want to ignore openssl validation